changelog/_unreleased/2024-10-15-theme-script-with-configuration-load-improved.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Use AbstractConfigLoader component to load configuration inside ThemeScript component
+issue: NEXT-00000
+author: Raffaele Carelle
+author_email: raffaele.carelle@gmail.com
+author_github: @raffaelecarelle
+---
+# Storefront
+* Changed `ThemeScript` component to use `AbstractConfigLoader`  to load theme configuration
```

Comment:
we should write the  FQCN and not just the name of the class. that will avoid confusion